website/docs/patterns/form-patterns/partials/validation/validation.md

```diff
@@ -59,6 +70,18 @@ Validating a form on submission can occur on the client or the server and can be
   <source src="/assets/patterns/form-patterns/validation-on-submit-example.mp4" />
 </video>
 
+### Delayed validation
+
+Delayed validation occurs on the client and refers to validating the field after a lapse in keystrokes and a specific interval of time, e.g., `500ms` or 0.5 seconds. Once the user has stopped input into the field and after the interval of time has expired, the field is validated without an `onblur` event occurring.
```

Comment:
Can't it be based on either, not both? 

```suggestion
Delayed validation occurs on the client and refers to validating the field after a lapse in keystrokes or a specific interval of time, e.g., `500ms` or 0.5 seconds. Once the user has stopped input into the field or after the interval of time has expired, the field is validated without an `onblur` event occurring.
```

Reply:
Yes, that's a good point, I'll update this!